### PR TITLE
Updated login/signup pages

### PR DESCRIPTION
**Pull request by krendell123**
I separated the code for login and sign-up pages into two HTML docs. Ideally I would like these to be through the same document and you switch between login and sign-up modes using Javascript but for now these are two separate documents.

I deleted the file login.js because it was not necessary with two different HTML documents.

Added form validation to sign-up fields. Will modify this when we agree on valid inputs (eg. password/username requirements).

Added a picture background. Still not sure if this is the kind of design we want but willing to keep experimenting with it!

### CODE REVIEW

File: js/signup.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                        message: 'Please supply a valid email address'
+                    }
+                }
+            },
+
+        	password: {
+                    validators: {
+                        identical: {
+                            field: 'confirmPassword',
+                            message: 'Confirm your password below - type same password please'
```
Comment (NickWu007) [inline js/signup.js:41]: I am not sure if this is the right field and message. Did you mix it with the one below?

File: login.html [hunk 2]
```diff
@@ -5,12 +5,16 @@
...
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+
     <!-- Bootstrap -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
 
     <!-- Custom styles for this template -->
-    <script src="login.js"></script>
+    <script src="js/login.js"></script>
```
Comment (NickWu007) [inline login.html]: I see you deleted login.js, but it doesn't seem like you have it anywhere in the codebase anymore. Is this intended?

Comment (krendell123) [reply login.html]: No sorry! When I separated the login and signup pages I moved the Javascript to the signup.js file and the login.js file was no longer necessary. I will probably add some client side functionality to this page later that will need js/login.js but for now I'll just delete it

**Review by NickWu007 — APPROVED** [review]

**Comment by krendell123** [comment]
I made some more changes to this branch

Created a randomised background mechanism
Restructured files to separate JS, CSS and HTML
Fixed buttons
Started on a basic logo design to go in top left corner

**Pull request merged**